File: Day_5/Q19.cpp
#include <iostream>
using namespace std;
//Printing The Factors Of The Number
void print_factors(int n){
    for(int i = 1;i*i <= n;i++){
        if(n % i == 0) {
            cout << i << "\t";
            if(i != (n/i)) cout << (n/i) << "\t";
        }
    }
}
int main(){
    int n;
    cin >> n;
    //Validating Input
    if(n <= 0) cout << "Invalid Input";
    else print_factors(n);
    return 0;
}

File: Day_5/Q19.java
import java.util.*;
class Q19{
    //Printing The Factors Of The Number
    static void print_factors(int n){
    for(int i = 1;i*i <= n;i++){
        if(n % i == 0) {
            System.out.print(i + "\t");
            if(i != (n/i)) System.out.print((n/i) + "\t");
        }
    }
}
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        //Validating Input
        if(n <= 0) System.out.print("Invalid Input");
        else print_factors(n);
    }
}